BREEZE.rst

```diff
@@ -1692,7 +1692,7 @@ This is the current syntax for  `./breeze <./breeze>`_:
 
                  wheel,sdist,both
 
-          Default: 
+          Default:
```

Comment:
@kaxil , this change (and the same one below) is breaking the static check (https://github.com/apache/airflow/pull/13379/checks?check_run_id=1625944061) . I'm not very sure why the pre-commit hook is enforcing a space here, but given it's a minor one, I suggest to revoke these two minor changes.

Reply:
done

Reply:
Yep, we spotted on it at the same time LOL